core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -1254,14 +1254,28 @@ public function purgeSearchIndex(DataContainer $dc)
 		$objResult = $this->Database->prepare("SELECT id FROM tl_search WHERE pid=?")
 									->execute($dc->id);
 
+		// TODO: Should we call contao.search.indexer.default delete() here instead of the duplicated code?
```

Comment:
Should we call contao.search.indexer.default delete() here instead of the duplicated code?